codebuild/release/version.yml

```diff
@@ -0,0 +1,27 @@
+version: 0.2
+
+env:
+  variables:
+    NODE_OPTIONS: "--max-old-space-size=4096"
+    BRANCH: "mainline-1.x"
+    # An explicit version bump
+    VERSION_BUMP: ""
+  git-credential-helper: yes
+
+phases:
+  install:
+    commands:
+      - npm ci --unsafe-perm
+    runtime-versions:
+      nodejs: 12
+  pre_build:
+    commands:
+      - git config --global user.name "aws-crypto-tools-ci-bot"
+      - git config --global user.email "no-reply@noemail.local"
+      - git checkout $BRANCH
+  build:
+    commands:
+      # Generate new version and CHANGELOG entry and push it
+      - npx lerna version --conventional-commits --git-remote origin --yes ${VERSION_BUMP:+$VERSION_BUMP --force-publish}
+      # Log the commit for posterity
+      - git log -n 1
```

Comment:
were you able to test this on your local to see if it worked?

Reply:
Not yet

Reply:
testing, on my local:
```
nvm use 12; npm run clean; rm -rf node_modules; npm ci --unsafe-perm;
git checkout -b 1.x-test-learna-version
git push --set-upstream public_tony 1.x-test-learna-version
npx lerna version --conventional-commits --git-remote origin --yes ${VERSION_BUMP:+$VERSION_BUMP --force-publish}
<generate GitHub auth token>
<export GitHub auth token as GITHUB_PAT>
gh auth login
npx lerna version --conventional-commits --git-remote public_tony --yes ${"":+"" --force-publish}
```
Results are here https://github.com/texastony/aws-encryption-sdk-javascript/commit/03424c569532c82c1735fc1c7da1bfc85892cfc0